packages/rsbuild-plugin-angular/src/lib/plugin/models/compiler-plugin-options.d.ts

```diff
@@ -22,7 +22,7 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-import { SourceFileCache } from './source-file-cache';
+import { SourceFileCache } from '../utils/source-file-cache';
```

Comment:
I assume this was a left over?
<img width="542" alt="Screenshot 2025-01-12 at 22 52 57" src="https://github.com/user-attachments/assets/bdb858a8-1ed0-4c30-8172-c0ab28c2199e" />